README.adoc

```diff
@@ -319,6 +319,9 @@ A full Prometheus endpoint configuration can look like this:
     #password: your-pass
   collection_interval: 1m
   metrics:
+  - name: jvm_memory_pool_bytes_committed
+    id: jvm_memory_pool_bytes_committed_${pool}
+    type: gauge
```

Comment:
for prometheus endpoints, type is not needed and in fact not recommended (since HOSA will determine the type by looking at the endpoint's metadata in the #TYPE line).